targets/digispark-pro.json

```diff
@@ -0,0 +1,21 @@
+{
+	"inherits": ["avr"],
+	"llvm-target": "avr-atmel-none",
+	"cpu": "attiny167",
+	"build-tags": ["digispark_pro", "attiny167", "attiny"],
+	"cflags": [
+		"-mmcu=attiny167"
+	],
+	"ldflags": [
+		"-mmcu=avr35",
```

Comment:
Should this also be `"-mmcu=attiny167"` like the `cflags` argument above?

Reply:
Just saw https://github.com/tinygo-org/tinygo/pull/946/commits/2381cf9b661bcfc8187332216b2c88af53ba76b7 so probably I am incorrect? or maybe this should be `"-mmcu=avr25"`?

Reply:
Exactly, apparently things go wrong when you specify the exact device. You can get the value of `-mmcu=avr*` from here: https://gcc.gnu.org/onlinedocs/gcc/AVR-Options.html (it lists the attiny167 under avr35).